hooks/post-command

```diff
@@ -31,6 +31,14 @@ main() {
     exit 0
   fi
 
+  CODECOV_SHA=$(curl -s https://codecov.io/bash | shasum -a 1 | awk '{ print $1 }')
+  REMOTE_SHA=$(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/master/SHA1SUM | awk '{ print $1 }')
```

Comment:
We don't know how fast the code from `https://raw.githubusercontent.com/codecov/codecov-bash` is deployed to `https://codecov.io/bash`, so I'd expect some sha failures from time to time.

Reply:
Yeah I just thought about that, I can get it from the version tag